### ISSUE TITLE
Paired, but unavailable, bluetooth device blocks "Scann for BLE sensors"

### ISSUE THREAD

**Issue by tekaeren** [bailuk/AAT#70]
Using latest v.1.16.1 release from F-Droid.

I've Amazfit Bip paired with my phone. When the watch is not connected (not in bluetooth range at all) "scan for BLE sensors" does the following:
- lists Amazfit Bip Watch Sensor with scanning mode and stays that way
- after starting another scan (or scans depends...) process finds another sensors that are in range, but also have problems in finishing scanning and marking these as "supported" and vailable for use.

It seems that AAT tries to connect in background to unavailable, but paired device wich affect bluetooth connectivity.

**Comment by arsab** [comment]
Actually Amazfit Bip is not supported it does claim to have Heart Rate 
But does not follow the standard.

**Comment by tekaeren** [comment]
Yes, I know - I do not try to connect to it, AAT just starts to scan the BIP entity automatically, even when it's not in range. Probably it lists devices already paired in Android bluetooth stack and tries to use them.
